DOCS-11426-aggregators-reference-kt (#1564)
mulesoft/docs-connectors: aggregators/1.0/modules/ROOT/pages/index.adoc

= Aggregators Module 1.0 - Mule 4
:page-aliases: connectors::aggregator/aggregators-module.adoc

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

For compatibility information and fixed issues, see the Aggregator Module Release Notes.

> Aggregators Module contains different types of aggregators capable of storing values and releasing them all together based on the configuration specified. An aggregator scope is a component used to collect those values until a condition is met, then execute a processor’s chain with the list of aggregated elements as the payload.

Reviewer: Is it "Aggregator" or "Aggregators" Module? Search for and correct GLOBALLY.

Reviewer: Alternative:
```suggestion
Aggregator Module contains different types of aggregators that use a specified configuration to both store and release values as a group. An _aggregator scope_ is a component that collects those values until a specified condition is met, when it then executes a processor’s chain of components, with the list of aggregated elements as the payload.
```

The basic aggregator configuration consists of the definition of the value to aggregate (extracted from the message through an expression evaluation), a condition that defines when the aggregation is considered complete, and a processor chain that executes once that condition is met.

> Aggregators are pass-through routers. The same data that an aggregator scope receives is processed by the components that follow the aggregator scope. The only modifications are the propagation of variables in case any are set while executing any of the aggregation routes.

Reviewer: Is this also correct (and simpler)?
```suggestion
Aggregators are pass-through routers. The same data that an aggregator scope receives is processed by the components that follow the aggregator scope, plus any variables set during the execution of any of the aggregation routes.
```
